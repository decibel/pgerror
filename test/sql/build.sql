\set ECHO none

\i test/pgxntool/psql.sql

BEGIN;
\i sql/pgerror.sql

\d error_data
\df try

\echo # TRANSACTION INTENTIONALLY LEFT OPEN!

-- vi: expandtab sw=2 ts=2
