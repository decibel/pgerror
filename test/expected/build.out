\set ECHO none
 Composite type "public.error_data"
     Column      | Type | Modifiers 
-----------------+------+-----------
 sqlstate        | text | 
 message         | text | 
 hint            | text | 
 detail          | text | 
 context         | text | 
 schema_name     | text | 
 table_name      | text | 
 column_name     | text | 
 constraint_name | text | 
 type_name       | text | 

                                         List of functions
 Schema | Name | Result data type |                  Argument data types                   |  Type  
--------+------+------------------+--------------------------------------------------------+--------
 public | try  | record           | code text, OUT row_count integer, OUT error error_data | normal
(1 row)

# TRANSACTION INTENTIONALLY LEFT OPEN!
